ui/raidboss/data/triggers/o11s.js

```diff
@@ -1,13 +1,63 @@
+
 'use strict';
 
 // O11S - Alphascape 3.0 Savage
 [{
   zoneRegex: /^Alphascape V3.0 \(Savage\)$/,
   timelineFile: 'o11s.txt',
+  timelineTriggers: [
+    {
+      id: 'O11S Reset',
+      regex: /Reset/,
+      beforeSeconds: 6,
+      alertText: {
+        en: 'get out, center aoe',
+      },
+    },
+    {
+      id: 'O11S Reset',
+      regex: /Reformat/,
+      beforeSeconds: 6,
+      alertText: {
+        en: 'donut aoe',
+      },
+    },
+    {
+      id: 'O11S Delta Attack',
+      regex: /:325B:Omega readies Delta Attack./,
+      beforeSeconds: 5,
+      alertText: function(data) {
+        if (data.role == 'tank')
+          en: 'TANK LB';
+      },
+      groupTTS: {
+        en: 'Tank LB',
+      },
+    },
+    {
+      id: 'O11S Ballistic Puddles',
+      regex: /Ballistic Impact/,
+      beforeSeconds: 7,
+      infoText: {
+        en: 'position for puddles',
+      },
+      groupTTS: {
+        en: 'drop puddles',
+      },
+    },
+    {
+      id: 'O11S Ballistic Stacks',
+      regex: /Engage Ballistic Systems/,
+      beforeSeconds: 3,
+      infoText: {
+        en: 'party stack',
+      },
+    },
+  ],
   triggers: [
     {
       id: 'O11S Mustard Bomb',
-      regex: / 14:326D:Omega starts using Mustard Bomb on (\y{Name})/,
+      regex: /14:326D:Omega starts using Mustard Bomb on (\y{Name})./,
```

Comment:
I think this works with the space.  Also, if you're going to add a `.` it should be a `\.` because it's regex.